frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ServingRuntimeTemplateSection.tsx

```diff
@@ -77,6 +89,23 @@ const ServingRuntimeTemplateSection: React.FC<ServingRuntimeTemplateSectionProps
           setData('servingRuntimeTemplateName', name);
         }}
       />
+      {data.servingRuntimeTemplateName && onFocusChange && (
+        <FormHelperText>
+          <HelperText>
+            <HelperTextItem>
+              You can optimize model performance by{' '}
+              <Button
+                isInline
+                onClick={() => onFocusChange('servingRuntimeArgsInput')}
+                variant="link"
+              >
+                configuring the parameter
```

Comment:
@yih-wang Is the microcopy for this ticket done? Should this be "configuring the parameter" or "configuring the parameters"

Reply:
We have "Configuration parameters" in the `FormSection`

Reply:
Oops, just realized I replied in the main thread below... 😅 I should not multitask during meetings next time.